decidim-generators/Gemfile.lock

```diff
@@ -32,8 +32,8 @@ PATH
       devise-i18n (~> 1.2)
       devise_invitable (~> 2.0)
     decidim-api (0.28.0.dev)
-      graphql (~> 1.12, < 1.13)
-      graphql-docs (~> 2.1.0)
+      graphql (~> 2.0)
```

Comment:
The CHANGELOG is listing multiple "breaking changes" in versions 1.13.0, 1.13.9, 1.13.10, 2.0.1, 2.0.16, 2.0.17 and 2.0.18. Also the 2.0.0 CHANGELOG states:
> [...] legacy code components were removed, so if there are any more references to those, there will be name errors!

So I think this would deserve a note in the RELEASE_NOTES.md?

Reply:
The changes were added in d8493953f3d434e610b18c7365272e87ecc0998f